src/main/kotlin/com/ritense/iko/ApplicationConfig.kt

```diff
@@ -19,4 +25,15 @@ class ApplicationConfig() {
         return camelContext.createProducerTemplate()
     }
 
+    @Bean
+    fun taskExecutor(): DelegatingSecurityContextAsyncTaskExecutor {
+        val executor = ThreadPoolTaskExecutor().apply {
+            corePoolSize = 50
+            maxPoolSize = 100
+            threadNamePrefix = "async-executor-"
```

Comment:
Maybe rename the threadNamePrefix to just task-exector. The async is kinda redundant imo.